src/envoy/http/authn/jwt_authn_utils.cc

```diff
@@ -0,0 +1,49 @@
+/* Copyright 2018 Istio Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "src/envoy/http/authn/jwt_authn_utils.h"
+
+namespace Envoy {
+namespace Http {
+namespace Istio {
+namespace AuthN {
+
+namespace {
+const std::string kJwtClusterName("example_issuer");
+}  // namespace
+
+// Get the Jwks URI for Envoy cluster
+const std::string getJwksUriEnvoyCluster() {
+  // Todo: Pilot needs to put a cluster field in the Istio authn JWT config.
```

Comment:
use TODO(lei-tang): instead of Todo. here and all

Reply:
Done.